### PR TITLE
Add page titles

### LINKED ISSUE #140: Set page titles
https://stackoverflow.com/a/54112771

### PR DESCRIPTION
**Pull request by kylesferrazza**
closes #140

### CODE REVIEW

File: app/javascript/components/common/documentTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  title: string;
+}> = (props) => {
+  const {
+    title,
+    children,
+  } = props;
+  useEffect(() => {
+    document.title = title;
+  }, []);
+  return <>{children}</>;
```
Comment (blerner) [inline app/javascript/components/common/documentTitle.tsx:13]: Is the `<> ... </>` necessary, or is the `children` fragment already of the right type?

File: app/javascript/components/workflows/index.tsx [hunk 2]
```diff
@@ -134,14 +135,16 @@ const Exam: React.FC = () => {
...
       return (
-        <ShowExam
-          railsUser={railsUser}
-          railsExam={showRes.response.railsExam}
-          railsCourse={showRes.response.railsCourse}
-          railsRegistration={showRes.response.railsRegistration}
-          final={showRes.response.final}
-          lastSnapshot={showRes.response.lastSnapshot}
-        />
+        <DocumentTitle title={showRes.response.railsExam.name}>
```
Comment (blerner) [inline app/javascript/components/workflows/index.tsx:138]: Does it matter whether `<DocumentTitle>` surrounds its children, or is just the first child of the `<Route>` before any others?

File: app/javascript/components/common/documentTitle.tsx (hunk 1 above)
Comment (kylesferrazza) [reply app/javascript/components/common/documentTitle.tsx:13]: https://github.com/DefinitelyTyped/DefinitelyTyped/issues/18051

File: app/javascript/components/workflows/index.tsx (hunk 2 above)
Comment (kylesferrazza) [reply app/javascript/components/workflows/index.tsx:138]: Either should work.

File: app/javascript/components/common/documentTitle.tsx (hunk 1 above)
Comment (blerner) [reply app/javascript/components/common/documentTitle.tsx:13]: :cry:

**Pull request merged**